src/service/api/article/article.e2e.test.js

```diff
@@ -0,0 +1,477 @@
+    "category": [
+      `Без рамки`,
+      `За жизнь`,
+      `Программирование`,
+      `Кино`
+    ],
+    "comments": [
+      {
+        "id": `yrODG`,
+        "text": `Планируете записать видосик на эту тему`
+      },
+      {
+        "id": `iGUeK`,
+        "text": `Давно не пользуюсь стационарными компьютерами. Ноутбуки победили.`
+      },
+      {
+        "id": `fevl0`,
+        "text": `Плюсую, но слишком много буквы!`
+      }
+    ]
+  },
+  {
+    "id": `3H9ok`,
+    "title": `Лучшие рок-музыканты 20-века`,
+    "createdDate": `2021-10-30 02:27:23`,
+    "announce": `Он написал больше 30 хитов,Помните, небольшое количество ежедневных упражнений лучше, чем один раз, но много,Ёлки — это не просто красивое дерево. Это прочная древесина,Из под его пера вышло 8 платиновых альбомов,Бороться с прокрастинацией несложно. Просто действуйте. Маленькими шагами`,
+    "fullText": `Достичь успеха помогут ежедневные повторения,Игры и программирование разные вещи. Не стоит идти в программисты, если вам нравятся только игры,Он написал больше 30 хитов,Программировать не настолько сложно, как об этом говорят`,
+    "category": [
+      `Программирование`
+    ],
+    "comments": [
+      {
+        "id": `WOpz5`,
+        "text": `Давно не пользуюсь стационарными компьютерами. Ноутбуки победили.`
+      },
+      {
+        "id": `Eb8fs`,
+        "text": `Планируете записать видосик на эту тему`
+      },
+      {
+        "id": `RfPr0`,
+        "text": `Хочу такую же футболку :-)`
+      }
+    ]
+  }
+];
+
+// INIT Function
+const createAPI = () => {
+  const app = express();
+  const cloneData = JSON.parse(JSON.stringify(mockData));
+
+  app.use(express.json());
+
+  article(app, dataService(cloneData));
+
+  return app;
+};
+
+
+// eslint-disable-next-line no-undef
+describe(`API returns a list of all articles`, () => {
+  const app = createAPI();
+
+  let response;
+
+  // eslint-disable-next-line no-undef
+  beforeAll(async () => {
+    response = await request(app).get(`/articles`);
+  });
+
+  // eslint-disable-next-line no-undef
+  test(`Status code 200`, () => {
+    // eslint-disable-next-line no-undef
+    expect(response.statusCode).toBe(HTTP_STATUS_CODE.OK);
+  });
+
+  // eslint-disable-next-line no-undef
+  test(`Returns a list of 5 offers`, () => {
+    // eslint-disable-next-line no-undef
+    expect(response.body.length).toBe(6);
+  });
+
+  // eslint-disable-next-line no-undef
+  test(`First offer's id equals "bUAlOA"`, () => {
```

Comment:
```suggestion
  test(`First article id equals "bUAlOA"`, () => {
```
замени везде слово `offer`